provider/uprops/src/lib.rs

```diff
@@ -0,0 +1,8 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+mod binary;
+mod upropdump_serde;
+
```

Comment:
Nit: Module-level docs (which will also become the README)

Reply:
Spent a while poking at this. Am I correct in concluding that the README is created / committed as part of a CI task, or do I need to take some concrete action beyond adding the module-level docs?

Reply:
After writing the docs in *lib.rs*, run `cargo make generate-readmes` to convert it to README.md.  The CI will check that you ran the command and checked in the artifact (so long as the README file is present).